### PR TITLE
Consolidate meta news and persian test files

### LINKED ISSUE #2216: Consolidate article tests all into one file (ati, body, meta and amp)
**Is your feature request related to a problem? Please describe.**
Consolidate articles tests in one spec file for all services

**Describe the solution you'd like**
Factor out test helpers into commands and use these in the single spec file

**Describe alternatives you've considered**
None, but I wouldn't try to do this in one PR

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by PriyaKR**
Resolves #2216 

**Overall change:** _Combined meta news and persian cypress test files into one generic `metaArticleSpec.js`also this would extend the meta feature to other services too when they are produced_

**Code changes:**

- _Deleted meta news and persian test file and created one generic `metaArticleSpec.js`_
- _Using `Object.keys` the config file `services.js` is traversed through all the article page available for different services(for now news and persian)_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ export default {
   igbo: {
     pageTypes: {
       articles: undefined,
+      fontStartsWith: undefined,
```
Comment (jamesbhobbs) [inline cypress/support/config/services.js]: This should be one level up in the config, it shouldn't be under pageTypes

Comment (PriyaKR) [reply cypress/support/config/services.js]: Done

**Comment by PriyaKR** [comment]
One thing i have noticed is running the test on local machine against `test` and `live` thtows accessing two domains issue as slready visiting the canonical page and then inside the tests the amp page is being visited too.

But not an issue on jenkins.This PR might fix this issue https://github.com/bbc/simorgh/issues/2123

File: cypress/integration/metaArticleSpec.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          lang: `${testData[service].datetimeLocale}`,
+          dir: `${testData[service].dir}`,
+        });
+      });
+
+      it('should have a nofollow meta tag', () => {
+        retrieveMetaDataContent('head meta[name="robots"]', 'noodp,noydir');
+      });
+
+      it('should load a maximum of two Reith font files', () => {
```
Comment (dr3) [inline cypress/integration/metaArticleSpec.js]: I dont think this test should exist. All it does is assert we have no more `Reith*` font families than `ReithSans` and `ReithSerif`. The only other is the `condensed` one, we dont need a whole test (and config) to ensure it doesn't load. (and this isnt even testing if fonts load, just if the font family is used in css)

Comment (dr3) [inline cypress/integration/metaArticleSpec.js:45]: I dont think this test should exist. All it does is assert we have no more Reith* font families than ReithSans and ReithSerif. The only other is the condensed one, we dont need a whole test (and config) to ensure it doesn't load. (and this isnt even testing if fonts load, just if the font family is used in css)

**Review by dr3 — CHANGES_REQUESTED** [review]

File: cypress/integration/metaArticleSpec.js (hunk 2 above)
Comment (PriyaKR) [reply cypress/integration/metaArticleSpec.js:45]: I haven't modified anything here these tests were existing earlier for news articles pages what i have done is scaled it for persian article page to look for Nassim font.

**Review by dr3 — APPROVED** [review]

**Review by jroebu14 — APPROVED** [review]

File: cypress/integration/metaArticleSpec.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        const fontFamiliesArray = [];
+        console.log(`${services[service].font}`);
```
Comment (thekp) [inline cypress/integration/metaArticleSpec.js]: you don't need this console log here ^

Comment (PriyaKR) [reply cypress/integration/metaArticleSpec.js]: removed it.

**Review by thekp — APPROVED** [review]
👍

**Comment by david-boydell** [comment]
Looks good @PriyaKR,  this is ready for merge 👍

**Pull request merged**